app/plugins/NasAnalysis/main.mi2app

```diff
@@ -3,10 +3,30 @@ import os
 import sys
 
 # Import MobileInsight modules
-from mobile_insight.monitor import OnlineMonitor
+from android.broadcast import BroadcastReceiver
+from jnius import autoclass
 from mobile_insight.analyzer import LteNasAnalyzer, UmtsNasAnalyzer
+from mobile_insight.monitor import OnlineMonitor
 import mi2app_utils
 
+def on_broadcast(context, intent):
```

Comment:
should define basic functionalities in `analyzer`, and let the plugins to reload/override it.